ci_config.json

```diff
@@ -199,6 +199,17 @@
       "windows": false
     }
   },
+  "libglvnd": {
+    "_comment": "Windows needs pthreads. macOS/LLD does not support --version-script",
+    "build_on": {
+      "darwin": false,
+      "windows": false
+    },
+    "_comment": "GLX tests require an X server, which CI does not provide.",
+    "build_options": [
+      "libglvnd:glx=disabled"
+    ]
```

Comment:
glx is a pretty import feature of glvnd, I think we should install `xorg-dev` (there may be a few more dev files needed, but xorg-dev should be most of it) for build tests.

Reply:
The tests require an X server running though.

Reply:
Looking back through the comments it looks like the bigger problem is if the x server isn't running the tests fail... maybe update the comment to reflect that instead

Reply:
We can run tests with xvfb-run, maybe? Probably needs updates to sanity_checks.py though.

Reply:
Upstream issues are beyond the scope of this PR.

Reply:
How is that an upstream issue? The idea is to perform tests with `xvfb-run meson test -C builddir/`.

Reply:
hrm? how would that be achieved?

Reply:
That's why I said:
> Probably needs updates to sanity_checks.py though